modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -901,12 +901,26 @@ definition {
 	}
 
 	@description = "Verify that is possible to delete translations"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanDeleteTranslations {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanDeleteTranslations";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
-		// TODO CanDeleteTranslations pending implementation
+		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(translationOptionSelected = "en-us");
+
+		ProcessBuilderKaleoDesignerReact.viewTranslationLanguagePresentOnDropdownMenu(translationOption = "fr_FR");
+
+		ProcessBuilderKaleoDesignerReact.gotoManageTranslations(
+			deleteLanguage = "French",
+			done = "true",
+			searchLanguage = "French");
```

Comment:
Same thing as before, it's no clear to me that here you are going to the manage translations, deleting the French language and clicking on done. This steps could be done separately